cdap-cli/src/main/java/co/cask/cdap/cli/CLIConfig.java

```diff
@@ -201,7 +200,7 @@ private boolean saveAccessToken(AccessToken accessToken, String hostname) {
     File accessTokenFile = getAccessTokenFile(hostname);
 
     try {
-      if (accessTokenFile.createNewFile()) {
+      if (accessTokenFile.exists() || accessTokenFile.createNewFile()) {
```

Comment:
I don't think you need a condition here. If the file exists then `createNewFile` just returns False. So you can ignore the value of that call. :

```java
accessTokenFile.createNewFile();
Files.write(accessToken.getValue(), accessTokenFile, Charsets.UTF_8);
return true;
```